src/main/java/seedu/uninurse/model/task/TaskList.java

```diff
@@ -157,26 +166,35 @@ public List<ListModificationPair> getDiff(GenericList<Task> otherTaskList) {
     }
 
     /**
-     * Returns a list of {@code Task}s that are due on a given day.
+     * Returns a list of Tasks that are due on a given day.
+     *
+     * @param day to get the Tasks on.
+     * @return The List of Tasks on the given Day.
      */
-    public List<Task> getAllTasksOnDay(DateTime test) {
-        return internalTaskList.stream().filter(t -> t.isTaskOnDay(test)).collect(Collectors.toList());
+    public List<Task> getAllTasksOnDay(DateTime day) {
+        return internalTaskList.stream().filter(t -> t.isTaskOnDay(day)).collect(Collectors.toList());
     }
 
     /**
-     * Returns a list of {@code Task}s that are due today.
+     * @return The List of Tasks that are for today.
      */
     public List<Task> getAllTasksToday() {
         return internalTaskList.stream().filter(Task::isTaskToday).collect(Collectors.toList());
     }
 
     /**
-     * Returns true if there are tasks today.
+     * @return If the TaskList contains any Tasks that are for today.
      */
     public boolean containsTaskToday() {
         return !this.getAllTasksToday().isEmpty();
     }
 
+    /**
+     * Checks whether the TaskList has the given Task.
+     *
+     * @param task to check for.
+     * @return Whether the given Task is in TaskList.
+     */
     public boolean hasTask(Task task) {
         return internalTaskList.contains(task);
     }
```

Comment:
I think this method should be removed as we would not want to provide an explicit way to check if `TaskList` contains a certain `Task` as no other objects would need to know this information.

Reply:
Sure that makes sense